index.html

```diff
@@ -3,10 +3,13 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css" /> 
+   <link href="style.css" rel="stylesheet" type="text/css" />
     <title>Document</title>
 </head>
 <body>
     <script src="script.js"></script>
+<h1>John Doe</h1>
+<h5>Software Developer, United States</h5>
+<p>Hello I'm John. I'm a Software Developer with expeience in UI/UX design. I create websites that help business and organizations to expand their clientel and meet their needs. I manage everything from web design and layout to their web hosting and security. My main languages are of course HTML/CSS as well as Javascript and REACT. </p>
```

Comment:
Since this will be in the body we should wrap this in a `<main>` tag [HTML Semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) Also, we should watch out for indentation. Since we installed prettier, a lot of it can be done if you format the doc w/ prettier (You can also format on save)